src/components/InstructorPage.js

```diff
@@ -0,0 +1,257 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+
+import Button from '@material-ui/core/Button'
+
+import './InstructorPage.css'
+
+//Services
+import peerReviewService from '../services/peerReview'
+
+//Actions
+//import instructorPageActions from '../reducers/actions/instructorPageActions'
+
+const GroupDetails = ({ myGroup }) => {
+  if (!myGroup) {
```

Comment:
What is this here for? On line 48 student names are passed to this function. So if there are no student names, who is the student that does not belong to the group? Instructor?

Reply:
Fixed the misleading text.